modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -319,6 +319,16 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
     });
   };
 
+  // TODO: TINY-9142: Remove this to make nested noneditable formatting work
+  const targetNode = FormatUtils.isNode(node) ? node : selection.getNode();
+  if (dom.getContentEditable(targetNode) === 'false' && !FormatUtils.isWrappableNoneditable(ed, targetNode)) {
+    // node variable is used by other functions above in the same scope so need to set it here
+    node = targetNode;
+    applyNodeStyle(formatList, node);
```

Comment:
What is `applyNodeStyle` for here? Aren't we just skipping formatting this node altogether? Or is the behaviour a lot more nuanced than that? What is the desired behaviour? 

* we can format the contenteditable false itself because we are able to toggle off that format
* we can't format the contenteditable false because it doesn't have the wrapper class ?

Reply:
Or is it because the applying of styling will put something _outside_ that we still need to apply the styles? If so, what's the significance of the wrapper class? What difference in behaviour does it create?

Reply:
To try and help answer this since Luke is pretty busy atm, selector formats can be configured to format the CEF element regardless by adding `ceFalseOverride: true` to the format. So it's that case whereby we still need to apply formats and `applyNodeStyle` handles that case and returns otherwise.

Reply:
Oh, I see. Each format has its own `ceFalseOverride` setting, so the checking is done in applyNodeStyle

```
 // Check if the node is nonediatble and if the format can override noneditable node
      if (dom.getContentEditable(node) === 'false' && !format.ceFalseOverride) {
        return true;
      }
```